client/src/bin/greengrass-cli

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Why not `/bin/sh`

Reply:
https://intellipaat.com/community/2637/why-do-people-write-the-usr-bin-env-python-shebang-on-the-first-line-of-a-python-script